src/dmd/root/rootobject.d

```diff
@@ -62,6 +62,14 @@ extern (C++) class RootObject
         assert(0);
     }
 
+    ///
+    extern(D) const(char)[] toString()
+    {
+        import core.stdc.string : strlen;
+        auto p = this.toChars();
+        return p[0 .. strlen(p)];
```

Comment:
Could use `toDString` from above.

Reply:
Here I choose not to, as we want `rootobject` to depends on as little as possible.